users/routes.js

```diff
@@ -51,9 +47,33 @@ function UserRoutes(app) {
   const account = async (req, res) => {
     res.json(req.session["currentUser"]);
   };
+  const addToRecentlyViewed = async (req, res) => {
+    if (req.session["currentUser"]) {
+      const { pokemonId } = req.params;
+      const recentlyViewed = req.session["currentUser"].recentlyViewed;
+      // add the new pokemon and removes any duplicates
+      const updatedViewed = recentlyViewed.filter((pok) => pok !== pokemonId);
```

Comment:
good feature

i currently have a case where bulbasaur appears twice in the last searched, see if you can replicate it (i'm running [web 15](https://github.com/vitzlo/rocket-mart-web-app/pull/15/) locally for reference)
> log into seller
> bulbasaur appears immediately in the last searched, probably due to profile listings
> look up a random pokemon
> look up bulbasaur, should appear twice in the list